pkgs/by-name/ta/taterclient-ddnet/package.nix

```diff
@@ -34,18 +34,18 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "taterclient-ddnet";
-  version = "9.0.0";
+  version = "9.0.1";
 
   src = fetchFromGitHub {
     owner = "sjrc6";
     repo = "taterclient-ddnet";
-    rev = finalAttrs.version;
-    hash = "sha256-2vWZ5RE65nJrKEtqD2/vK0RKnIK3mSYdlcS/OD9jFvw=";
+    rev = "refs/tags/V${finalAttrs.version}";
```

Comment:
i can see that we're being more specific with defining `refs/tags` - but is there an explicit reason to use it?

we're targetting the specific rev properly with `finalAttrs.version` already, aren't we?

i'm not against it, just curious.

Reply:
![image](https://github.com/user-attachments/assets/6d307eea-4c33-413b-ba00-7cd567320b86)

that explains it

darn you tater!!!

Reply:
> i can see that we're being more specific with defining `refs/tags` - but is there an explicit reason to use it?
> 
> we're targetting the specific rev properly with `finalAttrs.version` already, aren't we?
> 
> i'm not against it, just curious.

Using `refs/tags` allows you to explicitly use Git tags, otherwise it might target a branch with that name.